docs/content/dev/documentation.md

```diff
@@ -4,18 +4,19 @@ Our documentation is build with [mkdocs](https://www.mkdocs.org).
 
 ## Writing
 
-All documentation files are found in the `docs` directory of the [hedgedoc/hedgedoc repo](https://github.com/hedgedoc/hedgedoc). These files are just normal markdown files with nothing special about them.
+All documentation files are found in the `docs/content` directory of the [hedgedoc/hedgedoc repo](https://github.com/hedgedoc/hedgedoc). These files are just normal markdown files with nothing special about them.
 
-The config file for mkdocs lies in the root of the repo in a file called `mkdocs.yml`. With that file the theme and menu - amoung others - can be configured.
+The configuration for mkdocs lies in the `docs` folder in a file called `mkdocs.yml`. With that file the theme and menu - amoung others - can be configured.
 **Please note:** Any new files need to be linked to by other files or put in the navigation or the files will be very hard to find on the documentation website.
 
 ## Building
 
 To build the documentation locally you need to perform the following steps:
 
-1. make sure you have python3 installed
+0. make sure you have python3 installed
+1. Go into the `docs` folder
 2. install all the dependencies possibly with a [venv](https://docs.python.org/3/library/venv.html):  
-`pip install -r mkdocs.requirements.txt`
+`pip install -r requirements.txt`
 3. start the mkdocs dev server `mkdocs serve` or let it build the documentation `mkdocs build`
```

Comment:
As we have full sentences in this document and not only bullet-points, I'd suggest capitalizing the first letter of each point and end it with a dot.